### PR TITLE
Add new content for mutated insects

### PR DESCRIPTION
**Pull request by ymber**
#### Summary
SUMMARY: Content "Make bugs much more mutated"

#### Purpose of change
Resurrect #37158.

#### Describe the solution
Add harvest messages and new items, use snippets.

#### Testing
Game loads correctly. New content is present and described as expected.

### CODE REVIEW

File: data/json/items/comestibles/carnivore.json [hunk 1]
```diff
@@ -631,7 +657,7 @@
     "type": "COMESTIBLE",
     "copy-from": "fat",
     "name": { "str": "chunk of mutant fat", "str_pl": "chunks of mutant fat" },
-    "description": "Freshly butchered fat from a heavily mutated animal.  You could eat it raw, but it is better used as an ingredient in other foods or projects.",
+    "description": "Freshly butchered fat from a heavily mutated animal.  It smells, if anything, even more disgusting than the rest of the mutant.  There are little puddles of undentified oils dripping from it.",
```
Comment (LaVeyanFiend) [inline data/json/items/comestibles/carnivore.json:660]:
```suggestion
    "description": "Freshly butchered fat from a heavily mutated animal.  It smells, if anything, even more disgusting than the rest of the mutant.  There are little puddles of unidentified oils dripping from it.",
```

File: data/json/snippets/mutant_anatomy.json [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+        "id": "bug_organs_1",
+        "text": "These organs came from a giant mutant bug, and you really aren't sure what to make of them.  There are things you've never seen in any anatomy book, with spines and hair and other unidentified parts protruding off seemingly at random."
+      },
+      {
+        "id": "bug_organs_2",
+        "text": "These organs came from a giant mutant bug.  They have a sickly green color, and one of them ripped when you were removing it, revealing an inner surface that looks like row upon row of human fingers, nails and all."
+      },
+      {
+        "id": "bug_organs_3",
+        "text": "This is a huge, thick, fleshy sac you removed from a giant mutant bug.  THe surface is covered in smooth, soft skin, and beneath it is a coiled, twisted mess of cordlike tissue."
```
Comment (LaVeyanFiend) [inline data/json/snippets/mutant_anatomy.json:60]:
```suggestion
        "text": "This is a huge, thick, fleshy sac you removed from a giant mutant bug.  The surface is covered in smooth, soft skin, and beneath it is a coiled, twisted mess of cordlike tissue."
```

File: data/json/snippets/mutant_anatomy.json [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "<bug_harvest_acid_deeper>",
+    "text": [
+      "<bug_harvest_general_deeper>",
+      "The organs themselves have an acrid odour, but don't seem as caustically acidic as the outer shell.",
+      "You carefully avoid breaking through pockets of what you think may be acid-secreting glands.",
+      "Thick, ropey cords of tissue beneath its chitin protect an inner layer of strange organs, resembling those of a bird more than anything"
```
Comment (LaVeyanFiend) [inline data/json/snippets/mutant_anatomy.json:170]: I assume the second highlighted line needs a period

File: data/json/items/comestibles/carnivore.json [hunk 4]
```diff
@@ -230,7 +231,8 @@
     "type": "COMESTIBLE",
     "copy-from": "mutant_meat",
     "name": "cooked mutant meat",
-    "description": "This is a cooked chunk of meat from a mutated animal.  It has an unsettling, spongy texture, but otherwise tastes… mostly normal.  Hopefully you got all the bits of hair and bone out…",
+    "snippet_category": "cooked_mutant_meat_desc",
+    "description": "This is a cooked chunk of meat from a mutated animal.",
```
Comment (AlexMooney) [inline data/json/items/comestibles/carnivore.json:235]: This description only restates the item name.  The old description _described_ the object.  Think about how the meat would look, feel, or smell different.

Comment (ymber) [reply data/json/items/comestibles/carnivore.json:235]: The `cooked_mutant_meat_desc` strings are substituted into the description.

**Comment by kevingranade** [comment]
Check tests and clang-tidy results, this just needs some polishing.

File: data/json/items/comestibles/carnivore.json [hunk 5]
```diff
@@ -273,11 +276,22 @@
     "fun": -5,
     "flags": [ "EATEN_HOT" ]
   },
+  {
+    "id": "mutant_bug_organs",
+    "type": "COMESTIBLE",
+    "copy-from": "offal",
+    "name": { "str": "mutant organs", "str_pl": "mutant organs" },
```
Comment (kevingranade) [inline data/json/items/comestibles/carnivore.json:283]:
```suggestion
    "name": { "str_sp": "mutant organs" },
```

File: data/json/items/comestibles/carnivore.json [hunk 6]
```diff
@@ -432,6 +446,18 @@
     "parasites": 0,
     "delete": { "flags": [ "RAW" ] }
   },
+  {
+    "id": "mutant_bug_lungs",
+    "type": "COMESTIBLE",
+    "copy-from": "lung",
+    "name": { "str": "mutant lungs", "str_pl": "mutant lungs" },
```
Comment (kevingranade) [inline data/json/items/comestibles/carnivore.json:453]:
```suggestion
    "name": { "str_sp": "mutant lungs" },
```

File: data/json/items/comestibles/carnivore.json [hunk 7]
```diff
@@ -273,11 +276,22 @@
...
+    "looks_like": "offal",
+    "proportional": { "price": 0.1, "calories": 0.65 },
+    "extend": { "flags": [ "BAD_TASTE" ], "vitamins": [ [ "mutant_toxin", 25 ] ] }
+  },
   {
     "id": "offal_pickled",
     "copy-from": "offal",
     "type": "COMESTIBLE",
-    "name": { "str_sp": "pickled offal" },
+    "name": { "str": "pickled offal", "str_pl": "pickled offal" },
```
Comment (kevingranade) [inline data/json/items/comestibles/carnivore.json:294]:
```suggestion
    "name": { "str_pl": "pickled offal" },
```

File: data/json/items/comestibles/carnivore.json [hunk 8]
```diff
@@ -273,11 +276,22 @@
...
+    "looks_like": "offal",
+    "proportional": { "price": 0.1, "calories": 0.65 },
+    "extend": { "flags": [ "BAD_TASTE" ], "vitamins": [ [ "mutant_toxin", 25 ] ] }
+  },
   {
     "id": "offal_pickled",
     "copy-from": "offal",
     "type": "COMESTIBLE",
-    "name": { "str_sp": "pickled offal" },
+    "name": { "str_pl": "pickled offal" },
```
Comment (kevingranade) [inline data/json/items/comestibles/carnivore.json:294]:
```suggestion
    "name": { "str_sp": "pickled offal" },
```

**Comment by I-am-Erk** [comment]
Thanks for that!

**Comment by kevingranade** [comment]
This pull request has been mentioned on **Cataclysm: Dark Days Ahead**. There might be relevant details there:

https://discourse.cataclysmdda.org/t/gas-sacs-and-why-no-they-wouldnt-work/25566/2

**Pull request merged**